vllm/worker/model_runner.py

```diff
@@ -125,11 +125,14 @@ def _prepare_prompt(
             prompt_len = len(prompt_tokens)
             prompt_lens.append(prompt_len)
             prefix_len = 0
-            prefix = seq_group_metadata.prefix
-            if prefix is not None and prefix.computed:
-                prefix_len = prefix.get_length()
+
+            # NOTE: This only works for oooooooxxx style attention.
```

Comment:
can you link to the issue/RFC that describes what this is or future work should add?
```suggestion
            # NOTE: This only works for oooooooxxx style attention, where we need to compute attention on all "x" tokens.
```